### PR TITLE
apbiology: fix table width padding exercises apbiology

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #1675

### CODE REVIEW

File: styles/designs/carnival/parts/_eoc-components.scss [hunk 1]
```diff
@@ -134,4 +134,11 @@ $Exercise_Table: (
...
 );
-
+$Exercise_Table_Row: (
+    _name: "ExerciseTableRow",
+    _subselector: " .os-problem-container table > tbody > tr > td",
+    _properties:(
+        text-align:  enum('ValueSet:::OPTIONAL'),
+        padding: enum('ValueSet:::OPTIONAL'),
+    )
+);
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_eoc-components.scss]: In line 129 there is component `$Exercise_Table` with `_subselector: " .os-problem-container table",`. What do you think about making `$Exercise_Table_Row` its component? I mean to set in `$Exercise_Table_Row` subselector to:  `_subselector: "  > tbody > tr > td",` and then in the shape you have modified, do something like that:
```
map-merge( $Problem_container,(
    _components: (
         $Problem_Number,
         $Problem_Divider,
         $Exercise_Figure,
         map-merge( $Exercise_Table,(
            _components: (
                $Exercise_Table_Row,
            )
        ))
     )
 ))
```

File: styles/designs/carnival/parts/_eoc-shapes.scss [hunk 2]
```diff
@@ -93,6 +93,7 @@
                     $Problem_Divider,
                     $Exercise_Figure,
                     $Exercise_Table,
+                    $Exercise_Table_Row,
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_eoc-shapes.scss]: See my previous comment

File: styles/templates/template1/parts/_eoc-settings.scss [hunk 3]
```diff
@@ -304,6 +312,13 @@
             table-layout: fixed,
             width: 100%,
         ),
+        ExerciseTableHeadCell: (
+            padding: v-grid(.5) h-grid(.5),
+        ),
```
Comment (wilkus27) [inline styles/templates/template1/parts/_eoc-settings.scss]: I can't find this component in `eoc-components.scss` file. Is it created?

File: styles/designs/carnival/parts/_eoc-components.scss (hunk 1 above)
Comment (ajluk) [reply styles/designs/carnival/parts/_eoc-components.scss]: I have nested components as suggested. The code is for your review.

File: styles/templates/template1/parts/_eoc-settings.scss (hunk 3 above)
Comment (ajluk) [reply styles/templates/template1/parts/_eoc-settings.scss]: The unused settings has been deleted. The code is ready for your review.

**Pull request merged**